packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -601,6 +603,9 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 			];
 
 			const editors = await (window as any).tinymce.init({
+				content_style: `* {font-size: ${props.fontSize}px; font-family: ${props.fontFamily}; line-height: ${theme.lineHeight};}
```

Comment:
```suggestion
				content_style: `body {font-size: ${props.fontSize}px; font-family: ${props.fontFamily}; line-height: ${theme.lineHeight};}
```

Rather than selecting `*`, to prevent this from also affecting Mermaid blocks and plugin-generated output, consider instead styling `body`. If `body` isn't specific enough (doesn't override the existing font style on `body`), consider using `body#tinymce` or `#rendered-md`.  (From Joplin's development tools, it looks like the current `font-family` styles apply to the `body` selector).

Thank you for working on this!

Reply:
Ok np, this of course would be a lot better.
Also the macOS M1 build test failed, When I looked into the logs it says that the electron downloading step got `502` which stands for a `Bad Gateway` which means there was an invalid response from the server. Should I do something about it?

Reply:
As you predicted, `body` does not override the existing font style on `body`. But `body#tinymce` and `#rendered-md` do.
But before committing this change, should the table text `font-family` be overridden or not? because for now it doesn't.

![RTE](https://github.com/laurent22/joplin/assets/113056556/c3399395-d043-4494-9869-350fc76b6fb6)

Reply:
Yes it should

Reply:
> Also the macOS M1 build test failed, When I looked into the logs it says that the electron downloading step got 502 which stands for a Bad Gateway which means there was an invalid response from the server. Should I do something about it?

That looks like a random test failure. If the test passes after pushing other commits, it should be safe to ignore.

Reply:
After some searching, I found that the table's style is generated by TinyMCE and to change it we have to override it. So the solution I came up with is as follows:
```suggestion
				content_style: `body#tinymce {font-size: ${props.fontSize}px; font-family: ${props.fontFamily}; line-height: ${theme.lineHeight};} table.mce-item-table * {font-family: ${props.fontFamily} !important;}`,
```

<br>

- I did override only the `font-family` because the table doesn't have `line-height` property by default and the `font-size` property has an invalid value so it's ignored.

![table-style](https://github.com/laurent22/joplin/assets/113056556/c01e298b-d2c0-421e-9330-05e71edb043f)

Reply:
It looks like the original table style is applied here:

https://github.com/laurent22/joplin/blob/0715340dc0437efe785b227c2ea61820728b0485/packages/renderer/noteStyle.ts#L285

As such, an alternate solution might be to add an option to [Options](https://github.com/laurent22/joplin/blob/0715340dc0437efe785b227c2ea61820728b0485/packages/renderer/noteStyle.ts#L51) to configure the renderer font style. (This might be more complicated than overriding the existing styles though).